flake.lock

```diff
@@ -3,11 +3,11 @@
     "CHaP": {
       "flake": false,
       "locked": {
-        "lastModified": 1669917887,
-        "narHash": "sha256-9bsEaFh2lb26dZNUL+P4/LIzkTZH5NC7n9SprKzB/2A=",
+        "lastModified": 1677577017,
+        "narHash": "sha256-xS5EwzCI1WzZfdlf26zZ+QC+RAzU1FoACL/q4ibgHSg=",
         "owner": "input-output-hk",
         "repo": "cardano-haskell-packages",
-        "rev": "85510200dd0dc758d72bc1ada11ff5855e5d46b7",
+        "rev": "2d6658e30409bfd5f7bc76b23b3fa6b4198bccd6",
```

Comment:
You're having trouble here because you did `--update-input CHaP` like you're supposed to, but that got broken by https://github.com/input-output-hk/plutus-apps/pull/970/files#r1121538038